test/service_main_test.cc

```diff
@@ -0,0 +1,53 @@
+#include <grpc++/grpc++.h>
+
+#include "nighthawk/common/exception.h"
+
+#include "client/service_main.h"
+
+#include "gtest/gtest.h"
+
+using namespace testing;
+
+namespace Nighthawk {
+namespace Client {
+
+class ServiceMainTest : public Test {};
+
+// TODO(oschaaf): this gets us some coverage, but we need more functional testing.
+// See if we can add some python integration tests.
+TEST_F(ServiceMainTest, HelloWorld) {
+  std::vector<const char*> argv;
```

Comment:
I think you can use an initializer list rather than `push_back`.

Reply:
Fixed